### DISCUSSION TITLE
Eventhandling in OMNET++

### DISCUSSION THREAD

**Discussion by Abh4git** [omnetpp/omnetpp#1291] [Q&A]
I am seeing in my Event Log 
Log Event #121 ...............feedback_pkt coming. 
But I am not able to identify the method which is handling it. How can I trace the source and destination of an event? What is the best approach to identify? Any hints or pointers would be very helpful. I am trying to make some modifications for wireless. Thank you!

**Comment by rhornig** [comment]
you can inspect any event/message be slectingthe event and the looking at the properties in the lower left window in qtenv. The upper left window (and also the event timeline right below the toolbar buttons) can be used to find/select an event by clicking on the given event.

**Comment by Abh4git** [comment]
I now realize the fact that the event was getting handled correctly, but it was not showing up in logs. So I think I will close this one.
